src/forms/FormStateApp.tsx

```diff
@@ -22,29 +22,40 @@ import {
   FieldGroup,
   FormDivider,
   StaticField,
+  SubmitButton,
 } from "src/forms";
 import { BoundCheckboxGroupField } from "src/forms/BoundCheckboxGroupField";
 import { BoundTreeSelectField } from "src/forms/BoundTreeSelectField";
 import { FormLines } from "src/forms/FormLines";
-import { AuthorInput } from "src/forms/formStateDomain";
+import { AuthorInput, jan1 } from "src/forms/formStateDomain";
 import { useComputed } from "src/hooks";
 import { CheckboxGroupItemOption, NestedOption } from "src/inputs";
 import { HasIdAndName } from "src/types";
 
 export function FormStateApp() {
-  // Simulate getting the initial form state back from a server call
-  const queryResponse = useMemo(
```

Comment:
This moved down a few lines, and has a `isNew` conditional, so reformatted, but it's largely the same